skills/metabolomics/metabolomics-de/references/output_contract.md

```
@@ -0,0 +1,25 @@
+## Output Structure
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)